docs/src/docs/asciidoc/user/xml.adoc

```diff
@@ -35,6 +35,21 @@ These `Service` instances can then be used by `Cache` instances managed by the `
 extension point of the XML configuration (and Ehcache 3's modular architecture), as explained in the
 link:107{outfilesuffix}#_supplement_jsr_107_s_configurations[JSR-107 configuration section].
 
+=== `<default-serializers>` element
+
+A `<default-serializers>` element represents a collection of `Serializers` configured at `CacheManager` level. 
+Requires serializer `type` and a fully qualified class name of the `Serializer`.   
```

Comment:
Prefix with "Eache serializer ..." to indicate that the requires applies to the elements of the collection. Applies to copiers next